### ISSUE TITLE
precompiled version for Windows Release: sep 13 2015  - a bug with images

### ISSUE THREAD

**Issue by katarnad** [aardappel/treesheets#59]
At first some real big applause for this piece of a fine software.

The first time i go through a Tutorial for a Software and had even fun with it.
i've reopened the tutorial like over 10 times because i playfully discovered treesheets.
i've had every lesson new ideas for the usability.

in 2 days i used it for 5 different "projects". The main project was not direct to build.

Some little functions i had to add with my knowledge of autohotkey ( yep only a scripting language).
Know i've had endless possibilities.

**So now for the bug:**
I useing the version from the title of this message.

If pictures from the integrated dropdown or as external files inserted in an cell 
and someone folding with F10 the pictures are deleted/no visible after the unfolding.

**This makes the whole Imagesystem useless.**

On question also.
I'am not a fully programmer for c/net/delphi or something of the big one's
when makes it sense to compiling a newer version,if available ,bcause i must have a real good
reason to go throug the comile process on my own.  Is their a changelog for the source?

Then thanks already for this Versionof treesheets.

Knows the developer that he creates a new category of software.?

i searched something like a mindmaping tool or infobase, and got so much more.

- A  compare/analyze tool

- A  todo list tool

- A  manager an holder for my sets of Autoremote/Eventghost -system
the command getting/conversion was done with AHK, In Future Releases a lightweight
Option for Run with F4 on every Cell with variable input/output

- A  "mindalignment" tool - because the aligned cell part...who needs mindmap. :-).

- A a dataflatter

- A new problems to solve new tasks for treesheets....

**Comment by aardappel** [comment]
Yup, that's a known problem, it uses the image slot for the fold icon. I should really get around fixing that, which is a bit more work. Though if you're using folding a lot, you're probably not using TreeSheets to its full potential, shrinking is much more powerful.

The changelog is the commit log: https://github.com/aardappel/treesheets/commits/master

I guess this is a new kind of software :) Though it really is a blend between a spreadsheet and an outliner, I guess :)
